model-server/src/main/kotlin/org/modelix/model/server/ServerInitialization.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2024.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.modelix.model.server
+
+import org.modelix.model.server.store.IStoreClient
+import java.util.UUID
+
+class ServerInitialization(private val storeClient: IStoreClient) {
```

Comment:
Any reason this is a class and not a free function?

Reply:
Done. Made it a free function.

I had originally intended the class to have more fields and to grow to include more initialisation and migration code. But for now a free function is more appropriate.